src/index.js

```diff
@@ -1,13 +1,8 @@
-import _ from 'lodash';
 import './style.css';
-function component() {
-  const element = document.createElement('div');
-
- // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+import renderLayout from './layout.js';
 
-  return element;
+function component() {
```

Comment:
- [ ] From here onwards all projects will be strictly aligning with ES6 syntax. Writing functions is one of the things we check on. Let's refactor all functions to use ES6 syntax but use the arrow functions.